src/winter_onboarding_2021/fleet_management_service/handlers/cab.clj

```diff
@@ -26,9 +28,18 @@
           (merge success-flash (response/redirect "/cabs/new"))))))
 
 (defn new [request]
-  (response/response
-   (layout/application
-    request
-    "Add a cab"
-    (cab-views/cab-form
-     (get-in request [:flash :data])))))
+  (cab-views/cab-form
+   (get-in request [:flash :data])))
+
+(defn get-cabs [req]
+  (let [{:keys [page]} (:params req)
+        page-size (config/get-page-size)
+        page-int (Integer/parseInt (or page "1"))
+        offset (* page-size (- page-int 1)) ;;offset is 0 for for page 1
+        cabs (cab-model/select offset
+                               page-size)
+        rows-count (:count (first (cab-model/cabs-count)))
```

Comment:
The first & :count calls could move into the model so then `cabs-count` will return the count instead of a seq that has a map with the `:count` key and a corresponding count val.